.agent/skills/basic-git-conventions/SKILL.md

```
@@ -0,0 +1,186 @@
+```
+  tag v1.0.0        tag v1.1.0
+       │                 │
+───────┼─────────────────┼──── master (release-ready)
+       │        ↑        │
+       │     merge       │
+       │        │        │
+───────┼────────┼────────┼──── develop (integration)
+       │     ↑  ↑  ↑     │
+       │     │  │  │     │
+       │   feat fix chore│
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)